tests/data/snakeoil/export-a-surface

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
```

Comment:
The filename here: "export-a-surface" is not PEP8 (I think). Rename to "export_a_surface.py"?

If you try to import a file with a hyphen, like import my-file, Python will interpret the hyphen as a subtraction operation (i.e., my - file), which will result in an error.

Reply:
My thought was to have this as an "executable" (i.e. only `./export-a-surface` is intended) rather than a Python module. I'm not sure if that's the best choice and I don't have a strong preference... but for the forward model at least it needs to be chmod +x and with the shebang as such